lib/OpenQA/Schema/Result/JobGroupParents.pm

```diff
@@ -82,6 +82,11 @@ __PACKAGE__->add_columns(
     carry_over_bugrefs => {
         data_type   => 'boolean',
         is_nullable => 1,
+    },
+    build_view_mode => {
```

Comment:
Is this a leftover? Or do you intent to combine the cookie approach with a setting in the database? I personally would have preferred relying only on the cookie for now.